### ISSUE TITLE
Changing screen configure with configure virtual machine

### ISSUE THREAD

**Issue by bnice** [dirkwhoffmann/vAmiga#736]
Hi Dirk,

Just using vAmiga 2.1 . One thing that could be improved is not changing the screen colour when configure virtual machine is clicked. Why? Because when you are using the "Video" tab to adjust the screen, it would be helpful see how the controls are affecting the screen.

Regards,

Ben.

**Comment by dirkwhoffmann** [comment]
The darkening is done by the OS. It happens automatically when a sheet is displayed. But even without darkening it would not be optimal, because the sheet covers a large portion of the emulator texture. Maybe a preview image could be added to the video panel. I'll think about it...

**Comment by bnice** [comment]
Dam, sometimes the OS does the wrong the thing.

**Comment by dirkwhoffmann** [comment]
I've experimented with a couple of solutions:

- Solution 1: Add a preview image to the config panel. It turned out that the preview image is too small to get a good judgement about the color settings. Furthermore, the solution is technically difficult (e.g., the emulator has to run for two frames before the image can be taken). 

- Solution 2: Provide (optional) toolbar icons for adjusting brightness, contrast, and saturation. This approach works (it's possible to use circular sliders as toolbar items), but it looks ugly (e.g., the space between the sliders varies. It depends on the size of the toolbar labels).

Finally, I've decided to go for an approach which I've already used for the virtual keyboard. If the config panel is opened via the toolbar, it opens as a sheet (just as before). If the config panel is opened via the menu, the config panel is opened as a separate window. In this case, the emulator canvas is no longer darkened and fully visible.
